### PR TITLE
refactor!: renamed sound refice (reflected ice)

### PR DESCRIPTION
**Pull request by ghost**
Replace wSound with wRefIce (same with defense) as sound would behave the same as the wWhistle

### CODE REVIEW

File: src/base/zdefs.h [hunk 1]
```diff
@@ -765,7 +765,7 @@ enum
     wScript5, wScript6, wScript7, wScript8,
 	//39
     wScript9, wScript10, wIce, wFlame, //ice rod, fire rod
-    wSound, // -Z: sound + defence split == digdogger, sound + one hit kill == pols voice -Z
+    wRefIce, // -Z: sound + defence split == digdogger, sound + one hit kill == pols voice //merged into wWhistle... -J
```
Comment (connorjclark) [inline src/base/zdefs.h:768]: what does RefIce mean

**Comment by ghost** [comment]
Reflected IceMagic
________________________________

**Comment by ghost** [comment]
This should be ready to go

**Comment by connorjclark** [comment]
this change doesn't make any sense to me. is there any behavior change or just renaming things?

deferring to @EmilyV99

**Comment by EmilyV99** [comment]
> this change doesn't make any sense to me. is there any behavior change or just renaming things?
> 
> deferring to @EmilyV99

renaming unused things to other unused (but presumably to-be-implemented-soon) things

**Pull request closed without merging**